ci/build-package-docs.sh

```diff
@@ -55,6 +59,20 @@ go get -u github.com/pkg/errors
 # Regenerate the Node.JS SDK docs
 PKGS=${PKG_NAME} NOBUILD=true ./scripts/run_typedoc.sh
 
+# Regenerate the resource docs (for the specific plugin version) if applicable.
+case ${PKG_NAME} in
```

Comment:
I just chatted with @stack72 about `aiven` and `mailgun`. Those two are not yet ready for having the Pulumi schema generated. So we should exclude those two temporarily. Note how the `Makefile` for neither of them have a target for generating a schema yet (https://github.com/pulumi/pulumi-aiven/blob/master/Makefile and https://github.com/pulumi/pulumi-mailgun/blob/master/Makefile). I have this [issue](https://github.com/pulumi/docs/issues/2785) to track them when they are ready.

Reply:
Done.